test/nesting_test.rb

```diff
@@ -27,7 +26,7 @@
     assert_empty(impl.lock_stack)
   end
 
-  it 'does not raise errors with MySQL < 5.7.5 when acquiring nested error force enabled' do
+  test 'does not raise errors with MySQL < 5.7.5 when acquiring nested error force enabled' do
```

Comment:
This test should be removed if MySQL < 5.7.5 is unsupported (the `force...` param is removed because there's nothing to force anymore)